python/cudf/cudf/tests/test_list.py

```diff
@@ -332,3 +332,20 @@ def test_concatenate_list_with_nonlist():
         gdf1 = cudf.DataFrame({"A": [["a", "c"], ["b", "d"], ["c", "d"]]})
         gdf2 = cudf.DataFrame({"A": ["a", "b", "c"]})
         gdf1["A"] + gdf2["A"]
+
+
+@pytest.mark.parametrize(
+    "data",
+    [
+        [1],
+        [1, 2, 3],
+        [[1, 2, 3], [4, 5, 6]],
+        [None],
+        [1, None, 3],
+        [[1, None, 3], [None, 5, 6]],
+    ],
+)
+def test_list_getitem(data):
+    list_sr = cudf.Series([data])
+    # __getitem__ shall fill None with cudf.NA
+    assert pa.scalar(data) == list_sr.to_arrow()[0]
```

Comment:
Is this actually testing our `__getitem__` code path?

Reply:
oops :) fixed.